config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,13 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Does this file need to be included if it's all commented out?

Reply:
It's autogenerated and I prefer to keep the autogenerated files even if they're commented out.

It's brought our attention to one thing at least that we could consider valuable. =>

https://securityheaders.com/?q=https%3A%2F%2Fwww.apply-for-teacher-training.service.gov.uk%2F&followRedirects=on